tfjs-backend-webgpu/src/kernels/matmul_packed_webgpu.ts

```diff
@@ -193,7 +197,37 @@ export class MatMulPackedProgram implements WebGPUProgram {
             B[batch * ${batchBSize} + col * dimInner + row] : 0`;
     }
 
+    let activationSnippet = '', applyActivationSnippet = '';
```

Comment:
Any chance of sharing the duplicated (or mostly duplicated?) block here between the two implementations?

Reply:
Yes, it's a little noising that lots of code between them looks very similar and the include the code in conv2d. I will consider how to refactor this part and do it in a separate PR. Thanks.